todo.py

```diff
@@ -2,78 +2,66 @@ class TodoList:
     def __init__(self):
         self.tasks = []
 
-    def add_task(self, task, date = None):
-        """Adds a new task to the list if it doesn't already exist."""
+    def add_task(self, task, priority, date=None):
```

Comment:
Can you add a default to priority such as normal or medium? That way the user doesn't have to assign a priority for each task? Or have 3 options, low, medium, high?

Reply:
I think adding a default priority, such as "medium," is a great idea. It would streamline task creation and ensure that all tasks have an associated priority. I’ll try modifying the code so that when a user doesn't specify a priority, the task will automatically be assigned "medium" priority hopefully.